cartoframes/assets/templates/viz/main.html.j2

```diff
@@ -62,6 +62,9 @@
 <script>
   {% include 'utils/base64.js.j2' %}
 </script>
+<script>
+  {% include 'utils/html2canvas.js.j2' %}
```

Comment:
This is a huge file... It would be interesting to avoid using it when we are not going to create a static map. 

I know it is a pain to create another template, maintain a so on, but I would spend some time trying to make it conditional. I am mainly worried about Kuviz, where we are going to save and return all the HTML, passing from ~20kb to ~180kb

Reply:
You're 100% right, it's not a hard change, let's fix it 👍